README.md

```diff
@@ -80,6 +80,12 @@ A special `common` subsection under `backends` defines configuration settings th
           agent_name: agent01
           dry_run: false
           dry_run_output_dir: /opt/orb
+        otel:
+          grpc: "grpc://otel-collector:4317" # or `http: "http://otel-collector:4318"`
+          agent_labels:
```

Comment:
i would remove the http for now as well as it is not supported (only for pktvisor which is not documented yet)